### ISSUE TITLE
Can't find the option to remove an IP restriction

### ISSUE THREAD

**Issue by jmereaux** [mquekel/vsts-ip-restrictions#5]
Is such a functionality available?
Thanks

**Comment by Lutando** [comment]
Is there any plans to have a remove step?

**Comment by alexbf** [comment]
Your extension is great, a "remove step" to remove the Agent IP would be super useful!

**Comment by marrobi** [comment]
@mquekel this would be a very useful option.

**Comment by jamesfoster** [comment]
Agreed, the functionality to remove the build agent IP would be extremely useful. 

Also, according to the description of the plugin it allows you to "Add or remove IP restrictions".

https://github.com/mquekel/vsts-ip-restrictions/blob/master/vss-extension.json#L7
